### ISSUE TITLE
Usersnap Feedback - I made changes to this entry today. Al;though they were saved (1) my name [...]

### ISSUE THREAD

**Issue by jesicarson** [participedia/usersnaps#1470]
**Sender**: pscully@clearviewconsultingllc.com
**Comment**: I made changes to this entry today. Al;though they were saved (1) my name doesn't appear in the contributor history, and (2) it doesn't display on first page of search results. I don't care about my name appearing but something doesn't seem right about how the system is responding to my edits today.
[Open #1151 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-net-9cb6d041/1151)

<a href='https://usersnappublic.s3.amazonaws.com/2021-10-05/20-37/8c9f6c67-6364-4bbf-a1a9-0e89bac2005c.png'>![Usersnap Feedback - I made changes to this entry today. Al;though they were saved (1) my name (...)](https://usersnappublic.s3.amazonaws.com/2021-10-05/20-37/c5b91aae-c340-4231-9e92-1889133e5761.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2021-10-05/20-37/8c9f6c67-6364-4bbf-a1a9-0e89bac2005c.png'>Download original image</a>

**Browser**: Firefox 92 (Windows 10)
**Referer**: [https://participedia.net/case/7595](https://participedia.net/case/7595)
**Screen size**: 1760 x 1100 **Browser size**: 1760 x 939

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

--

A friendly reminder from Usersnap 👋

There’s a new widget in town. Make your bug reporting process sleeker and more advanced by changing to the new project template.

Learn more about how to transfer your project (and what new features you’ve been missing out on) in this [blog post](https://usersnap.com/blog/platform-new-widget/).

**Comment by jesicarson** [comment]
@plscully could you try logging out, refreshing your browser (or even restarting your machine), update your browser if that is available to you, and give the site another test?

**Comment by plscully** [comment]
@jesicarson I did all of this for both Firefox & Chrome.  It didn't work for either one and didn't save my edits. Very odd!

**Comment by jesicarson** [comment]
Thanks!

@paninee please have a look. Thanks!

**Comment by plscully** [comment]
@paninee @jesicarson It would be a big help if we could get this problem fixed asap.  I need to be able to make changes to PPedia entries that we are linking to in the series of essays that will be published in a few weeks. ... I tried to make a change to a difrerence case today https://participedia.net/case/63 but it didn't work. I tried using both Chrome & Firefox, both of which are updated. I also logged out and back in to my PPedia account.
